server/routes/conditions.ts

```diff
@@ -82,9 +80,9 @@ export default ({
       logger.debug('GET /additionalConditions')
 
       const { action, bookingId } = req.params
-      const licence = res.locals.licence?.licence
-      const bespokeConditions = licence?.licenceConditions?.bespoke || []
-      const conditions = conditionsServiceFactory.forVersion(CURRENT_CONDITION_VERSION).getAdditionalConditions(licence)
+      const record = res.locals.licence
+      const bespokeConditions = record.licence?.licenceConditions?.bespoke || []
+      const conditions = conditionsServiceFactory.forLicence(record).getAdditionalConditions(record.licence)
```

Comment:
rather than always using the current version, now read version from licence if possible. 
Only if that is not present then use the current version